components/match2/commons/match_summary_base.lua

```diff
@@ -673,7 +674,12 @@ function MatchSummary.defaultGetByMatchId(CustomMatchSummary, args, options)
 	local match, bracketResetMatch = MatchGroupUtil.fetchMatchForBracketDisplay(
 		args.bracketId, args.matchId)
 
-	local matchSummary = MatchSummary():init(options.width)
+	local width = options.width
```

Comment:
don't we need to Table.extract here so we do not get anno warnings when passing options to the other functions?

Reply:
No, passing additional values is always legal